analyzers/its/sources/SonarLintExclusions/SonarLintExclusions/NotIncluded.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SonarLintExclusions
+{
+    internal class NotIncluded
```

Comment:
Could you please align it with the other files? Remove the usings and add the  // S2094 comment